neurobooth_os/tasks/smooth_pursuit/pursuit_task.py

```diff
@@ -112,6 +117,10 @@ def run_trial(self, movement_pars):
         # Send a 'TRIAL_RESULT' message to mark the end of the trial
         self.sendMessage("TRIAL_RESULT")
 
+        # TODO: Should this go before the message sending above?
```

Comment:
Is the return needed here in the first place? The abort key seems to be breaking out of the loop which will execute the sendMessage lines and the task will finish after that any way. Am I understanding this correctly?

Reply:
> Is the return needed here in the first place? The abort key seems to be breaking out of the loop which will execute the sendMessage lines and the task will finish after that any way. Am I understanding this correctly?

Yeah, I had it up higher earlier. It's not needed unless we should skip some of the post-loop statements. Are all those messages supposed to be sent even if the task is aborted?